src/globals.ts

```diff
@@ -0,0 +1,29 @@
+import chokidar from 'chokidar';
+import chalk from 'chalk';
+import Command from '@oclif/command';
+
+export const GreenLog = chalk.hex('#00FF00');
+export const OrangeLog = chalk.hex('#FFA500');
+export const RedLog = chalk.hex('#FF0000');
+const CHOKIDAR_CONFIG = {
+  // awaitWriteFinish: true
+};
+export const WATCH_MESSAGES = {
+  logOnStart: (filePath: string) => console.log(GreenLog(`Watching AsyncAPI file at ${filePath}\n`)),
+  logOnChange: (handlerName: string) => console.log(OrangeLog(`Change detected, running ${handlerName}\n`)),
+  logOnAutoDisable: (docVersion: 'old' | 'new') => console.log(RedLog(`Watch mode for ${docVersion} file was not enabled..\n`), OrangeLog('INFO: Watch works only with files from local file system\n'))
+};
+
+export const specWatcher = (filePath: string, handler: Command, handlerName: string) => {
```

Comment:
should not logging of `logOnAutoDisable` actually be handled here?

Reply:
Yes @derberg  thanks for that refactor idea
I will try to refactor the above logic into specWatcher only.